src/open_inwoner/templates/pages/plans/create.html

```diff
@@ -50,13 +50,11 @@ <h1 class="h1">{% trans "Start nieuwe samenwerking" %}</h1>
                             <div class="modal modal--no-reset" id="template-{{ plan_template.id }}">
                                 <div class="modal__container">
                                     <h2 class="h2 modal__title" id="modal__title">{{ plan_template.name }}
-                                        {% button text=_("Sluiten") hide_text=True icon="close" extra_classes="modal__button modal__close-title" %}
+                                        {% button type="button" text=_("Sluiten") hide_text=True icon="close" extra_classes="modal__button modal__close-title" %}
                                     </h2>
                                     <div class="modal__text" id="modal__text">{{ plan_template.string_preview }}</div>
                                     <div class="modal__actions modal__actions--align-right" id="modal__actions">
-                                        {% spaceless %}
-                                            {% button text=_("Sluiten") extra_classes="modal__button modal__close" primary=True %}
-                                        {% endspaceless %}
+                                        <button class="button modal__button modal__close button--primary button-icon--primary" type="button" aria-label="Sluit"><span class="inner-text">Sluiten </span></button>
```

Comment:
I note this change uses plain html while the one above it uses the template tag (but I have no opinion on it)

Reply:
The one in the top above it is a standard cross icon that needs no visible text so can just use the template - but the one below is where things got complicated: the original Modal was overwriting all buttons _entirely_, instead of just overwriting their inner-most text, which meant it was impossible to give Modal submit-buttons etc. an icon; so that is one of the things this wall of a PR is tackling.